crates/iota-types/src/gas_model/gas_v1.rs

```diff
@@ -186,7 +186,9 @@ mod checked {
         // and then conceptually
         // `final_computation_cost = total_computation_cost * gas_price / reference_gas_price`
         gas_price: u64,
-        // RGP as defined in the protocol config.
+        // Reference gas price as defined in protocol config.
+        // If `protocol_defined_base_fee' is enabled, this is a mandatory base fee paid to the
+        // protocol.
```

Comment:
Nit: doc comments
```suggestion
        /// Reference gas price as defined in protocol config.
        /// If `protocol_defined_base_fee' is enabled, this is a mandatory base fee paid to the
        /// protocol.
```
You may want to adjust the rest of the documentation in this struct's fields.